django/core/management/commands/dumpdata.py

```diff
@@ -6,6 +7,39 @@
 from django.core.management.utils import parse_apps_and_model_labels
 from django.db import DEFAULT_DB_ALIAS, router
 
+try:
+    import bz2
+    has_bz2 = True
+except ImportError:
+    has_bz2 = False
+
+try:
+    import lzma
+    has_lzma = True
+    lzma_kwarg = {'format': lzma.FORMAT_ALONE}
+except ImportError:
+    has_lzma = False
+
+
+def get_open_method(filename, mode='wt'):
```

Comment:
I don't see a reason to include `mode` in this method, we always return the passed value.

Reply:
This method is used also in test with 'rt' mode to check the created fixtures.

Reply:
I know but it's not worth complexity, we can use
```python
stream = open_method(output, 'wt', **kwargs) if output else None
```
in `dumpdata` and
```python
with open_method(filename, 'rt') as f:
```
in tests.

Reply:
Fixed